### PR TITLE
Add ivs 0.1.0 post

### PR DESCRIPTION
**Pull request by DavisVaughan**
- [ ] Update date in the post

### CODE REVIEW

**Review by jennybc — APPROVED** [review]
Overall, LGTM!

I made a few comments.

Very high-level reaction: I feel like I want some pictures! I'm not sure if it's worth making diagrams just for the post, but if you've already made some for another purpose, that would be a reason to feature one example over another. I think the post will mostly be an awareness-raising advertisement for ivs, so that the folks with these interval problems remember to try out iv the next time it's relevant.

File: content/blog/ivs-0-1-0/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+or the `year_month_day` type from [clock](https://clock.r-lib.org):
+
+```{r}
+start <- year_month_day(c(2019, 2020), c(1, 3))
+end <- year_month_day(c(2020, 2020), c(2, 6))
+
+iv(start, end)
+```
+
+Interval vectors are always composed of right-open intervals, and the intervals must be strictly increasing.
```
[the remark this answers is not in the document]
Comment (DavisVaughan) [reply content/blog/ivs-0-1-0/index.Rmd:89]: Yea, this was very confusing, thanks!

**Comment by DavisVaughan** [comment]
I added a few images and I think they definitely improved the post, thanks!

File: content/blog/ivs-0-1-0/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+I say more on the practical reasons for this [in the Getting Started vignette](https://davisvaughan.github.io/ivs/articles/ivs.html#structure) if you are interested in learning more.
```
[the remark this answers is not in the document]
Comment (DavisVaughan) [reply content/blog/ivs-0-1-0/index.Rmd:90]: I'll talk to Hadley about this, currently unknown 😬

**Comment by jennybc** [comment]
Oh, yes, the diagrams make a huge difference. Very helpful!

**Comment by DavisVaughan** [comment]
Closing in favor of keeping this a Davis package and talking about it on my own blog https://github.com/DavisVaughan/blog/commit/869dd78edb913775a465ea739b609355da8cfd7d

**Pull request closed without merging**